week-3/E-array-map/exercise.js

```diff
@@ -2,4 +2,14 @@
 // Write multiple solutions using different syntax (as shown in the README)
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
+var newArray = [];
 
+function times10(number) {
+  return number * 100;
+}
+
+var numbers = numbers.map(function(e) {
```

Comment:
There is no need to create a new array.
```suggestion
var numbersMultiplied = numbers.map(function (number) {
    return number * 100;
});
```